module/documents/actor/actor.mjs

```diff
@@ -500,14 +500,24 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
         rollData.attributes.ac = ac;
         try {
-          const replaced = Roll.replaceFormulaData(formula, rollData);
-          ac.base = Roll.safeEval(replaced);
+          if ( game.release.generation < 12 ) {
+            const replaced = Roll.replaceFormulaData(formula, rollData);
+            ac.base = Roll.safeEval(replaced);
+          } else {
+            const roll = new Roll(formula, rollData);
```

Comment:
There's a small difference there, I think. The replacement done by `new Roll()` is done with `{ missing: "0" }`, which is not done in the V11 case.
`const roll = new Roll(Roll.replaceFormulaData(formula, rollData));` would be equivalent to the V11 case, but maybe `{ missing: "0" }` should have been used in the V11 case. I'm not sure.

Reply:
I think we don't want to do that so we can catch errors in AC formulas if they have missing data. But calling roll normally doesn't do that either, so maybe we'll have to use our custom `replaceFormulaData` method.